lib/streamlit/elements/multiselect.py

```diff
@@ -273,9 +276,6 @@ def serialize_multiselect(value: List[Any]) -> List[int]:
             multiselect_proto.set_value = True
 
         self.dg._enqueue("multiselect", multiselect_proto)
-        if len(widget_state.value) != 0:
```

Comment:
This is exactly what I was hoping to see :star_struck: 

I really appreciate your quick response to the concerns I had with this! :pray: